website/src/content/support-policy.mdx

```diff
@@ -0,0 +1,167 @@
+- React releases some new feature that requires a specific minimal version of React. Therefore, commercetools could release a new major version that requires custom applications to run to a minimal React version.
+
+</Info>
+
+All breaking changes are thoroughly documented in the release notes together with migration steps to facilitate the version update. This also helps to identify if changes are needed within the custom application's code.
+
+# Minimal required maintenance policy
+
+All the official Merchant Center applications are constantly kept up-to-date with the released tools and packages. This ensures that all the latest bug fixes and features are consistently adopted by all applications.
+
+In regards to custom applications, commercetools does not expect the same amount of maintenance as it requires internally.
+However, commercetools **recommends** that custom applications are **updated at regular periods** and **maintained in planned intervals**, to avoid falling too many versions behind.
+
+| Feature maturity     | Supported update interval                 |
+| -------------------- | ----------------------------------------- |
+| Beta                 | At least once a quarter (every 3 months)  |
+| General Availability | At least twice a year (every 6 months)    |
+
+<Info>
+
+It is important to keep the dependencies up-to-date, especially to get the latest bug fixes and security patches. See also [Automate maintenance of dependencies](#automate-maintenance-of-dependencies).
+
+</Info>
+
+## Deprecation notice and End Of Life (EOL)
+
+Some packages, for example `@commercetools-frontend/application-shell`, interact with different commercetools HTTP APIs. Changes to those packages might include updates related to HTTP API functionalities, usually in a backwards compatible way.
+However, it can happen that some HTTP API functionalities might be **deprecated** over time and **eventually removed**.
+
+<Info>
+
+commercetools guarantees that HTTP APIs do not have breaking changes, as they are not and will not be versioned.
+
+</Info>
+
+In case some HTTP API functionalities need to be deprecated, commercetools starts a **deprecation process** as following:
+
+1. The HTTP API functionality is marked as deprecated (for example GraphQL fields get the `@deprecated` directive)
+2. The deprecated functionality is removed from the packages that refer and use that functionality
+3. A new breaking change version of the packages is released, marking the starting point for how long the feature will remain deprecated
+4. The deprecation period (start to end) is documented on this page
+5. During the deprecation period, commercetools tracks the usage of the deprecated functionality, to determine if there are still applications using that functionality
+6. Towards the end of the deprecation period, commercetools might contact the owners/mainteners of applications that are still using versions prior to the deprecation release
```

Comment:
Yeah, sounds like a good idea. We need to prioritize this then.

Reply:
Contact E-Mails were mentioned before and are a good idea. However, we also indirectly have E-Mails as the project is owned by an organization which has an administrator team. It's not good but maybe good enough until we allow selecting a user of the admin (or even any) team of the owning org. Also a custom app might be developed by an external party and then plugged into the MC. Who should we contract then? The external party directly or some intermediary within the owning organization? In any case E-Mail field is good but we should talk to Cusomer Success if they see value in a seperate E-Mail or e.g. already have technical contact for each project/org.

Reply:
I think it makes sense to have it separated. Think about any marketplace app, like Github. If there is a problem you usually contact the developer of the app, not the admin of the github org. It might also be beneficial for the partners developing the apps, to bill the customer for maintenance updates 😅

Reply:
Yes and no. I don't want to hold hard opinions though. In your GitHub example you might want to contact the owner if you want to update a plan of the app. In case of a bug you contact the creator of the app. In case of CT you might want to go through the owner of the org as partners will want to contact owners before starting to work to request the budget. It's a bit workflow dependent which is why I think Customer Success might have good views towards this.

In general I think both needs exist. However, the need to contact the developer comes later when a single app is installed through the marketplace (as you mentioned) in multiple orgs/projects. Which is where we're not yet. So later we add a field "Contact developer" while now we can use the org's admin emails as most apps are devleoped close to them.

Reply:
Good points. Let's gather some feedback first and settle on the requirements. We can use the org admin's email in case we need to for now.